test/soliton.jl

```diff
@@ -0,0 +1,17 @@
+using Distributions
+
+K, M, δ, atol = 100, 60, 0.2, 0
```

Comment:
Can we put all these tests into one `@testset` like
```julia
@testset "Soliton" begin
    K, M, δ, atol = 100, 60, 0.2, 0
    ...
    @test cdf(Ω, K) ≈ 1
end
```

Reply:
I agree. I've made the change.